04/app.js

```diff
@@ -32,9 +40,21 @@ function getChannelColor(colorInHex, channelName) {
     }
 
     const channelColorHex = colorInHex.substr(start, 2);
+    // console.log(channelColorHex)
     const channelColorDec = parseInt(channelColorHex, 16);
 
     return channelColorDec; 
 }
 
+function changeBoxColor(e) {
+    setBoxShadow(boxElement, e.target.value);
+}
+
+function changeOpacity(e) {
+    const isMouseMoveEvent = e.type === 'mousemove';
+    const isMouseLeftButtonPress = e.buttons === 1;
+    if(isMouseMoveEvent && isMouseLeftButtonPres || !isMouseMoveEvent) {
+        boxElement.style.opacity = (rangeEl.value)/100;
```

Comment:
Spróbuj wykorzystać tutaj funkcje `setBoxShadow()` - zobacz, że trzeci parametr to opacity. ;)